### DISCUSSION TITLE
Project Beta: Hide Closed Issues

### DISCUSSION THREAD

**Discussion by wesleyhowery** [community/community#14451] [Projects and Issues]
Hi,

It would be nice to be able to hide closed issues on the list view on the new project beta. I find the tool to be very useful managing our workload but when the list gets longer it is annoying to have all the closed issues still showing up.

I do not want to delete them because in the board view I can see all the Done issues and be able to show our clients what we have completed and what is left. 

What do other people do to deal with their list view getting too crowded on the new project beta?

**Comment by rachelywong** [comment]
Also having this issue at the moment. I've found I can filter out "Status = Done" to move them to the bottom of the board, but then I'm unable to filter by priority or another column.

**Reply by dtzar** [reply]
Have you tried doing a `is:closed` in the search filter view?

**Comment by rachelywong** [comment]
it works! thank you so much 🙂
________________________________
